test.js

```diff
@@ -724,28 +723,6 @@ ltest('data and level-sublevel ttl meta data separation (custom ttlEncoding)', f
   })
 })
 
-// ltest('data and level-sublevel ttl meta data separation (custom sublevel encoding)', function (t, db, createReadStream) {
-//   var subDb = bwSublevel(db)
-//   var meta = subDb.sublevel('meta')
-//   var ttldb = ttl(db, { sub: meta, ttlEncoding: bytewise })
-//   var batch = randomPutBatch(5)
-
-//   ttldb.batch(batch, { ttl: 10000 }, function (err) {
-//     t.ok(!err, 'no error')
-//     db2arr(createReadStream, t, function (arr) {
-//       batch.forEach(function (item) {
-//         // bytewise keys in bytewise sublevels are double-encoded for now
-//         contains(t, arr, bwEncode([ [ 'meta' ], bwEncode([ item.key ]) ]), bwRange())
-//         contains(t, arr, {
-//           gt: bwEncode([ [ 'meta' ], bwEncode([ 'x', new Date(0), item.key ]) ]),
-//           lt: bwEncode([ [ 'meta' ], bwEncode([ 'x', new Date(9999999999999), item.key ]) ])
-//         }, bwEncode(item.key))
-//       })
-//       t.end()
-//     }, { keyEncoding: 'binary', valueEncoding: 'binary' })
-//   })
-// })
```

Comment:
This does no longer work since `level-ttl` batches meta data here https://github.com/Level/level-ttl/blob/4ada06a22c10aacb6c7108fc0b80cf2c8877f127/level-ttl.js#L113 and since we use `encoding-down`, the encoding will be `binary` for both keys and values.

The reason this worked before is because `level-sublevel` handles encoding itself, thus overriding `'binary'` with custom encoding (in this case `bytewise`)